orbs/jobs/run.yml

```diff
@@ -44,42 +48,45 @@ steps:
         LINK="https://bintray.com/codacy/Binaries/download_file?file_path=$CODACY_PLUGINS_TEST_VERSION%2Fcodacy-plugins-test-linux"
         wget -nc $LINK -O $FILENAME # FILENAME injected in .circle/config.yml
         chmod +x $FILENAME
-        docker load --input ~/workdir/docker-image.tar
-  - when:
-      condition: << parameters.run_json_tests >>
-      steps:
-        - run:
-            name: RunJsonTests
-            command: |
-              codacy-plugins-test/$FILENAME json << parameters.docker_image_name >>:<< parameters.docker_image_tag >>
-  - when:
-      condition: << parameters.run_pattern_tests >>
-      steps:
-        - run:
-            name: RunPatternTests
-            command: |
-              codacy-plugins-test/$FILENAME pattern << parameters.docker_image_name >>:<< parameters.docker_image_tag >>
-  - when:
-      condition: << parameters.run_plugin_tests >>
-      steps:
-        - run:
-            name: RunPluginTests
-            command: |
-              codacy-plugins-test/$FILENAME plugin << parameters.docker_image_name >>:<< parameters.docker_image_tag >>
+        docker load --input << parameters.docker_path >>
   - when:
       condition: << parameters.run_metrics_tests >>
       steps:
         - run:
             name: RunMetricsTests
             command: |
               codacy-plugins-test/$FILENAME metrics << parameters.docker_image_name >>:<< parameters.docker_image_tag >>
-  - when:
-      condition: << parameters.run_multiple_tests >>
+  - unless:
+      condition: << parameters.run_metrics_tests >>
       steps:
-        - run:
-            name: RunMultipleTests
-            command: |
-              codacy-plugins-test/$FILENAME multiple << parameters.docker_image_name >>:<< parameters.docker_image_tag >>
+        - when:
+            condition: << parameters.run_json_tests >>
+            steps:
+              - run:
+                  name: RunJsonTests
+                  command: |
+                    codacy-plugins-test/$FILENAME json << parameters.docker_image_name >>:<< parameters.docker_image_tag >>
+        - when:
+            condition: << parameters.run_pattern_tests >>
+            steps:
+              - run:
+                  name: RunPatternTests
+                  command: |
+                    codacy-plugins-test/$FILENAME pattern << parameters.docker_image_name >>:<< parameters.docker_image_tag >>
+        - when:
+            condition: << parameters.run_plugin_tests >>
+            steps:
+              - run:
+                  name: RunPluginTests
+                  command: |
+                    codacy-plugins-test/$FILENAME plugin << parameters.docker_image_name >>:<< parameters.docker_image_tag >>
+        - when:
```

Comment:
This is non-default too, Should it be using `unless` too ?

Reply:
everything is inside `unless` whill be ignored. Are we using `multiple` test on `metrics`? If not, I guess we should ignore that too when using `metrics` test .

Reply:
I did not get the indentation. So everything is inside the `run_metrics_tests`'s `unless`. Makes sense. 👍